src/generator/predefined/reducer.js

```diff
@@ -0,0 +1,74 @@
+const createTemplate = module.parent.createTemplate;
+
+const reducerTemplate = createTemplate`
+/* @flow */
+const INITIAL_STATE = null;
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    default:
+      return state;
+  }
+};
+`;
+
+const testTemplate = createTemplate`
+/*global describe it*/
+/*global expect*/
+import reducer from "${args => args.path}";
+
+describe("${args => args.name}", () => {
+  it("returns the initial state", () => {
+    const state = void 0;
+    expect(
+      reducer(state, {})
+    ).to.equal(null);
+  });
+});
+`;
+
+const newReducersIndexFactory = (name, path) => `
+import { combineReducers } from "redux";
+import ${name} from "${path}";
+
+export default combineReducers({
+  ${name}
+});
+`;
+
+module.exports = exports = options => ({
+  modify: {
+    file: "src/reducers/index",
+    modificator: content => {
```

Comment:
just mentioning one more time that I think all `modificator` should be `modifier`